plan/test_plan.py

```diff
@@ -628,9 +629,37 @@ def test_available_plans_for_sentry_customer_users_trial_plan_ongoing_trial_more
         expected_result.append(BASIC_PLAN)
         expected_result += PR_AUTHOR_PAID_USER_PLAN_REPRESENTATIONS.values()
         expected_result += SENTRY_PAID_USER_PLAN_REPRESENTATIONS.values()
+        expected_result += TEAM_PLAN_REPRESENTATIONS.values()
 
         assert plan_service.available_plans(owner=self.owner) == expected_result
 
+    def test_available_plans_for_more_than_10_activated_users(self):
```

Comment:
And can you indicate in the test title this is for trialing users too? `test_available_plans_for_trialing_users_with_more_than_10_activated_users` or something like that.

I'm okay with splitting this into two tests too, as this says more than 10 activated users, but you test a scenario with < 10 first, then > 10